addon/components/bs-accordion.js

```diff
@@ -2,7 +2,7 @@ import Ember from 'ember';
 import layout from '../templates/components/bs-accordion';
 import listenTo from '../utils/listen-to-cp';
 
-const { K: noop } = Ember;
+const {} = Ember;
```

Comment:
Interesting. I didn't consider this. Seems that the codemod should be smart enough to do that